user_guides/jsonp/src/main/jbake/content/title.adoc

```diff
@@ -17,7 +17,7 @@ Technology Compatibility Kit User's Guide for {TechnologyFullName}
 
 Release {TechnologyVersion} for Jakarta EE
 
-September 2019
+September 2020
```

Comment:
@edbratt Is the month September correct. I made the same for EL as per JakartaEE9 release. Should this be July/August when we intend to promote the TCK.